### ISSUE TITLE
youtube_trailer return encoded url

### ISSUE THREAD

**Issue by hoffmann2002** [opacam/python3-tmdb3#7]
if i want to get a trailerurl the "v" parameter is encoded and therefore incorrect.

here is an example:
```
x=TMDb.Movie(287947)
x.youtube_trailers[0].geturl()                                                                                                                                                    
"http://www.youtube.com/watch?v=b'go6GEIrcvFY'"
```

i think it has something to do with this code from class YoutubeTrailer:
```
self.source = self.source.encode("ascii", errors="ignore")
return f"http://www.youtube.com/watch?v={self.source}"
```

or did i used the function in a wrong way?

Thanks

**Comment by opacam** [comment]
Thanks for reporting this issue :smile:

This is fixed in master branch via PR #8

**Comment by hoffmann2002** [comment]
Thanks!
